js/ui/scheduler/shaders/ui.scheduler.current_time_shader.js

```diff
@@ -1,37 +1,44 @@
-const $ = require('../../../core/renderer');
-const Class = require('../../../core/class');
-const getBoundingRect = require('../../../core/utils/position').getBoundingRect;
+
+import { getBoundingRect } from '../../../core/utils/position';
+import $ from '../../../core/renderer';
 
 const DATE_TIME_SHADER_CLASS = 'dx-scheduler-date-time-shader';
 
-const currentTimeShader = Class.inherit({
-    render: function(workspace) {
-        this._workspace = workspace;
-        this._$container = workspace._dateTableScrollable.$content();
+class CurrentTimeShader {
+    constructor(workSpace) {
+        this._workSpace = workSpace;
+    }
+
+    render() {
+        this._$container = this._workSpace._dateTableScrollable.$content();
 
-        this._$shader = this._createShader();
+        this._$shader = this.createShader();
         this._shader = [];
```

Comment:
Just a suggestion.. To my mind it is better to rename `_shader` to `_shaders`, `_shaderStore`, `_shaderCollection` etc.